upstream-community-operators/hpa-operator/hpa.package.yaml

```diff
@@ -0,0 +1,5 @@
+#! package-manifest: deploy/chart/catalog_resources/rh-operators/hpaoperator.v0.1.6.clusterserviceversion.yaml
+packageName: hpa
```

Comment:
Please make sure the `packageName` and operator's directory match.
In this case, either change the `packageName` to `hpa-operator` or change the directory to `hpa`

Reply:
fixed, thx